GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -181,6 +182,30 @@ - (OIDAuthState *) authState{
   return ((GTMAppAuthFetcherAuthorization *)self.fetcherAuthorizer).authState;
 }
 
+#if TARGET_OS_IOS && !TARGET_OS_MACCATALYST
+
+- (void)addScopes:(NSArray<NSString *> *)scopes
+    presentingViewController:(UIViewController *)presentingViewController
+                  completion:(nullable void (^)(GIDUserAuth *_Nullable userAuth,
+                                                NSError *_Nullable error))completion {
+  [GIDSignIn.sharedInstance addScopes:scopes
```

Comment:
What happens if this instance of `GIDGoogleUser` isn't the same as `GIDSignIn.sharedInstance.currentUser`?